src/main/webapp/dev/js/test/CommonJsTest.es6

```diff
@@ -206,33 +206,37 @@ QUnit.test('setStatusMessage(message,status)', (assert) => {
     // isError = false: class = overflow-auto alert alert-warning
     // isError = true: class = overflow-auto alert alert-danger
 
+    function getClassesForStatusMessage(color) {
```

Comment:
The param name should be `statusType` instead. Also, since the function is defined inside a test about status messages, can remove `ForStatusMessage` from the function name. I suggest `getExpectedClasses(statusType)`.